### PR TITLE
guetzli 1.0 (new formula)

### PR DESCRIPTION
**Pull request by fkistner**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/guetzli.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  depends_on "gflags"
+
+  resource "test_image" do
+    url "https://github.com/google/guetzli/releases/download/v1.0/bees.png"
+    sha256 "2c1784bf4efb90c57f00a3ab4898ac8ec4784c60d7a0f70d2ba2c00af910520b"
+  end
+
+  def install
+    system "make"
+    bin.install "bin/Release/guetzli"
```
Comment (fxcoudert) [inline Formula/guetzli.rb]: No `make install`? No man page, no other shared files, no library, nothing?

Comment (fkistner) [inline Formula/guetzli.rb]: No `install` target, no man page, no library.
Just the command line reference implementation of [Google's new open-source JPEG encoder, Guetzli](https://research.googleblog.com/2017/03/announcing-guetzli-new-open-source-jpeg.html).
The repository also contains an undocumented [python script](https://github.com/google/guetzli/blob/master/tools/guetzli-compare.py) as well as a copy of the [butteraugli tool](https://github.com/google/butteraugli), which seem to have been used to create the comparison results for their paper. However, it seemed to me that neither are appropriate to be included in this formula.

**Review by fxcoudert — APPROVED** [review]

File: Formula/guetzli.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+class Guetzli < Formula
+  desc "Perceptual JPEG encoder"
+  homepage "https://github.com/google/guetzli"
+
```
Comment (dunn) [inline Formula/guetzli.rb]: No need for this blank line!

File: Formula/guetzli.rb (hunk 1 above)
Comment (dunn) [inline Formula/guetzli.rb]: Would you mind asking the developers to add a `make install`? Can't hurt.

**Comment by fxcoudert** [comment]
Thanks @fkistner for your contribution!

File: Formula/guetzli.rb (hunk 1 above)
Comment (fkistner) [inline Formula/guetzli.rb]: Issue has been filed upstream: google/guetzli#53

**Pull request closed without merging**